VAMobile/.detoxrc.json

```diff
@@ -28,7 +28,7 @@
       "type": "ios.simulator",
       "device": {
         "type": "iPhone 15 Pro Max",
-        "os": "iOS 17.4"
+        "os": "iOS 17.2"
```

Comment:
Did you intend to include this change?

Reply:
Just so there is a record.  Yes I did because the detox runs are still usually getting 1 or 2 timeouts for iOS.  I changed it to see if this works better.  I didn't have any timeouts on the detox run I did so I'm hopeful this will remove them all together.